frontend/components/carousel/component.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  FC,
+  useState,
+  useMemo,
+  Children,
+  cloneElement,
+  useEffect,
+  useRef,
+  useCallback,
+  useLayoutEffect,
+} from 'react';
+
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import { debounce } from 'lodash-es';
+
+import AriaLive from './aria-live';
+import Arrows from './arrows';
+import Paging from './paging';
+import { SlideProps } from './slide';
+import type { CarouselProps } from './types';
+
+export const Carousel: FC<CarouselProps> = ({ className, children }: CarouselProps) => {
```

Comment:
The arrows are not visible on narrow screens and the dots (pagination) looks a bit cramped.

Reply:
On iOS 12.5.5 (my iPhone), the swipe doesn't work on the slides though it works on newer iOS versions. I guess the problem is with the pointer events.

If you find an easy way to fix this, that'd be great (either by changing the events or having a  polyfill such as https://cdn.polyfill.io/v3/). If that's too much effort, we can forget about this issue since we have the arrows and dots.

Reply:
I hid the arrows for small screens. I'll give it a go at debugging it, but if it takes to long I can re-enable them (maybe move them to the bottom next to the dots.

Reply:
I couldn't find a great solution, so I've changed it so that the arrows are shown on small screens, next to the dots. 

I've tried a few polyfills (and made the component noSSR, but the behavior of the slider was sketchy to say the least. 

<img width="374" alt="Screenshot 2022-03-22 at 15 38 22" src="https://user-images.githubusercontent.com/6273795/159520941-28c74ee8-eb62-43f6-bcc4-d7d73a6552b5.png">

Reply:
That's good enough. It looks very nice 👏!